app/models/plan.rb

```diff
@@ -419,7 +418,7 @@ def owner
                   .administrator
                   .order(:created_at)
                   .pluck(:user_id).first
-    User.find(usr_id)
+    usr_id.present? ? User.find(usr_id) : nil
```

Comment:
was a bug in scenarios where the user_id (owner) is no longer attached to the plan

Reply:
Thanks.
I've noticed we do some inconsistent access around the display of a plan's owner due to the same thing